assets/js/night.js

```diff
@@ -17,12 +17,14 @@ function activateNightMode () {
 
 function deactivateNightMode () {
   body.removeClass(nightMode)
-  try { localStorage.removeItem(nightMode) } catch (e) { }
+  try { localStorage.setItem(nightMode, false) } catch (e) { }
```

Comment:
Is this change necessary or is it a style change?

Reply:
It's necessary to tell if a user intentionally set the theme to day time. That way we can prefer what the user has set, and fall back to what the browser wants.

I just realized this still has a bug, will fix.